puphpet/config.yaml

```diff
@@ -0,0 +1,204 @@
+---
+vagrantfile-local:
+    vm:
+        box: ubuntu-precise12042-x64-vbox43
+        box_url: 'http://box.puphpet.com/ubuntu-precise12042-x64-vbox43.box'
+        hostname: null
+        network:
+            private_network: 192.168.56.132
+            forwarded_port:
+                CocLsFE7aIwA:
+                    host: '8081'
+                    guest: '80'
+                CocLsFE7aIwE:
+                    host: '8083'
+                    guest: '3306'
+        provider:
+            virtualbox:
+                modifyvm:
+                    natdnshostresolver1: on
+                    memory: '1024'
+                setextradata:
+                    VBoxInternal2/SharedFoldersEnableSymlinksCreate/v-root: 1
+            lxc:
+                box: fgrehm/precise64-lxc
+                box_url: https://atlas.hashicorp.com/fgrehm/boxes/precise64-lxc/versions/1.2.0/providers/lxc.box
+        provision:
+            puppet:
+                manifests_path: puphpet/puppet
+                manifest_file: manifest.pp
+                options:
+                    - '--verbose'
+                    - '--hiera_config /vagrant/puphpet/puppet/hiera.yaml'
+                    - '--parser future'
+        synced_folder:
+            8Q9vTs08YuK2:
```

Comment:
There should also be an 'id' param like this:
```
             8Q9vTs08YuK2:
                id: 8Q9vTs08YuK2
                source: ./
```
Otherwise vagrant up fails on Windows.